sky/clouds/cloud.py

```diff
@@ -467,5 +468,11 @@ def check_disk_tier_enabled(cls, instance_type: str,
         """
         raise NotImplementedError
 
+    @classmethod
+    def query_status(cls, name: str, tag_filters: Dict[str, str],
+                     region: Optional[str], zone: Optional[str],
+                     **kwargs) -> List['status_lib.ClusterStatus']:
```

Comment:
Can we also add a short docstring here very briefly explaining what this method is supposed to do and what it returns? I was implementing this for Kubernetes last week and took some time to understand what needs to be done to implement this : )

Reply:
Good point! Just added. Thanks!